es-vue-base/src/lib-components/EsErrorPage.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+    <div class="my-5 container">
+        <div class="mb-5 mx-auto col-md-7 text-center">
+            <slot name="errorType" />
```

Comment:
While using slots is definitely the way to go for most design system components that have text/content like this, I think for the purpose of this it might make sense to use prop(s) instead.  The reason would be to avoid having to repeat this text on each repo this is dropped into, since the text and error types aren't likely to change.

To that end, I think probably only the error code itself would need to be a prop - The message texts themselves could just be data in this component, mapped to the error codes;  and if the input is an error code other than these 4, could map to a default generic msg.